test/e2e/features/smoke/phoneNumber.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Phone number test
+  @MWPW-144460 @smoke @geo-phoneNumber
+  Scenario Outline: Geo IP phone number
+    Given I go to the DC page '<urlPath>'
+    Then I record phone number on the page
+    Then I go to the page "<urlPath>" with geo-ip spoof "<countryCode>"
+    Then I confirm phone number is not changed
```

Comment:
Shouldn't the number be changed?

Reply:
Hi James, I've just updated a test, please have a look:
https://github.com/adobecom/dc/pull/621/commits/830ecb80ff42daf08f76020cee3a8e805eab9075